### ISSUE TITLE
Error when trying to run "npx co build"

### ISSUE THREAD

**Issue by snicol21** [drwpow/cobalt-ui#84]
I've gone through all of the instructions and am doing exactly what the original build suggests, however i'm getting this error:

`✘  Only URLs with a scheme in: file, data, and node are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'
 ELIFECYCLE  Command failed with exit code 1.`

Can't figure out how to resolve it.

**Comment by drwpow** [comment]
Sure! Can you post:

- Node version
- Operating System
- Your `tokens.config.js` file
- Your basic project file structure (e.g. where is the config, where is the tokens file, etc.)

**Comment by snicol21** [comment]
- Node version: v18.17.0
- Operating System: Microsoft Windows 10 Enterprise

- token.config.mjs
```
import pluginJS from "@cobalt-ui/plugin-js"

/** @type import('@cobalt-ui/core').Config */
export default {
  tokens: "./src/tokens.json",
  outDir: "./src/__generated__/",
  plugins: [pluginJS()],
}
```

- basic file structure:
```
.
├── pnpm-workspace.yaml
└── package.json/
    └── packages/
        └── tokens/
            ├── package.json
            ├── tokens.config.mjs
            └── src/
                └── tokens.json
```

**Comment by snicol21** [comment]
@drwpow I just upgraded to node version `20.5.1`, since v20 was recommended in the [Getting Started](https://cobalt-ui.pages.dev/docs/getting-started/) docs, however that didn't seem to fix the problem. So now i'm wondering if it's because i'm using `pnpm` as my package manager 🤔

**Comment by drwpow** [comment]
Good news! I just added Windows tests and was able to reproduce the bug. It should be fixed in `@cobalt-ui/cli@1.4.2`. Please let me know if you encounter any more issues 🙂

**Comment by snicol21** [comment]
Awesome!! Thanks @drwpow ... just tested it and it worked like a charm. Appreciate the help and the response.
